### ISSUE TITLE
Live region: aria-atomic does not work (IE 11, Chrome)

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/VFO-standards-support#290]
### Summary

Live region: aria-atomic does not work (IE 11, Chrome)

1. Save as HTML file:
```
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<title>live region</title>
	</head>
	<body>
		<div role=status><h4>heading1</h4><div>1<div id=0>change this</div>2</div>outside</div>
		<div role=status><h4>heading2</h4><div aria-atomic=true>3<div id=1>change this</div>4</div>outside</div>
		<div role=status aria-atomic=false><h4>heading3</h4><div>5<div id=2>change this</div>6</div>outside</div>
		<div role=status aria-atomic=false><h4>heading4</h4><div aria-atomic=true>7<div id=3>change this</div>8</div>outside</div>

		<h2>aria-atomic</h2>
		<button onClick="document.getElementById('0').innerHTML='changed 0';">none</button>	
		<button onClick="document.getElementById('1').innerHTML='changed 1';">true</button>	
		<button onClick="document.getElementById('2').innerHTML='changed 2';">false</button>	
		<button onClick="document.getElementById('3').innerHTML='changed 3';">false/true</button>	

	</body>
</html> 
```
2. Activate all buttons (the page must be reloaded after each heading)

### Expected result
* Consistent output between browsers
* output without aria-atomic = aria-atomic=false
* false: "Assistive technologies will present only the changed node or nodes"
* true: "Assistive technologies will present the entire changed region as a whole, including the author-defined label if one exists"

### Actual result
* Different output from browser to browser
* output only correct in Firefox
* No effect in IE 11 (aria-atomic only works if it is on the same element as the live region)
* Chrome: completely wrong output

see also: https://github.com/w3c/aria-practices/issues/78 and many other problems with live regions: https://github.com/FreedomScientific/VFO-standards-support/issues?q=live+region

The following overview shows which output is made:

#### IE 11
* none: "changed 0"
* true: "changed 1"
* false: "changed 2"
* false/true: "changed 3"

#### Firefox
* none: "changed 0"
* true: "3 changed 1 4"
* false: "changed 2"
* false/true: "7 changed 3 8"

#### Chrome
* none: all content of the live region
* true: all content of the live region
* false: "changed 2"
* false/true: "changed 3"

#### JAWS version and build number

JAWS 2019.1907.42
 
#### Operating System and version

Windows 8

#### Browser and version:

Chrome 76.0.3809.100
Firefox 68.0.2
Internet Explorer 11.0.9600.19431
